tests/testthat/test-spread.R

```diff
@@ -2,14 +2,14 @@ context("Spread")
 library(dplyr, warn.conflicts = FALSE)
 
 test_that("order doesn't matter", {
-  df1 <- data.frame(x = c("a", "b"), y = 1:2)
-  df2 <- data.frame(x = c("b", "a"), y = 2:1)
+  df1 <- tibble(x = c("a", "b"), y = 1:2)
```

Comment:
Changing from `data.frame()` to `tibble()` subtly changes the test because `tibble()` doesn't turn strings into factors.

Reply:
good point, added a factor() to all relevant places in test-spread